conf/db/migration/metrics_portal_ddl/common/V1__create_hosts_table.sql

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-CREATE TABLE portal.hosts (
+CREATE TABLE IF NOT EXISTS portal.hosts (
```

Comment:
You cannot make changes to a migration.  It will be considered invalid on all databases where it has been applied.